packages/updating-element/src/updating-element.ts

```diff
@@ -740,8 +740,13 @@ export abstract class UpdatingElement extends HTMLElement {
       // This `await` also ensures that property changes are batched.
       await this._updatePromise;
     } catch (e) {
-      // Ignore any previous errors. We only care that the previous cycle is
-      // done. Any error should have been handled in the previous update.
+      // Refire any previous errors async so they do not disrupt the update
+      // cycle. Errors are refired so developers have a chance to observe
+      // them, and this can be done by implementing
+      // `window.onunhandledrejection`.
+      Promise.resolve().then(() => {
```

Comment:
Can you just do `Promise.reject(e)`?

Both `Promise.resolve().then(...)` and `Promise.reject(e)` seem to have the same behavior in devtools, though neither is triggering `onunhandledrejection` or `onerror` for me. Might just be devtools weirdness.

Reply:
Good call, done.